examples/themes/netflix-theme.html

```diff
@@ -4,22 +4,149 @@
     <meta name="viewport" content="width=device-width" />
     <title>Media Chrome Netflix Theme Example</title>
     <script type="module" src="../../dist/index.js"></script>
-    <script type="module" src="../../src/js/themes/media-theme-netflix.js"></script>
+    <script type="module" src="../../src/js/theme/media-theme.js"></script>
   </head>
   <body>
     <style>
       body {
         margin: 0px;
       }
 
-      media-theme-netflix {
+      media-theme {
         width: 100vw;
         height: 100vh;
       }
+
+      video {
+        width: 100%;      /* prevents video to expand beyond its container */
+      }
     </style>
 
     <main aria-label="Media Chrome Netflix Theme Example">
-      <media-theme-netflix>
```

Comment:
Here's what I think we should do:
* Take the old media-theme.js and rename it media-theme-package.js (open to other names)
* Have that base MediaThemePackage class require that a `<template>` be provided in the init args. (This will help avoid the prior confusion of if a theme is a template or a web component)

With that, you can still build a `<media-theme-netflix>`, which would be a dumb wrapper to the template. You can also then load the netflix theme from `<media-theme>`, like:
```js
const mediaThemeEl = document.querySelector('media-theme')
mediaThemeEl.template = mediaThemeNetflix.template
```